src/Yasumi/Provider/Switzerland/Geneva.php

```diff
@@ -91,6 +92,7 @@ private function calculateJeuneGenevois(): void
         }
 
         $this->addHoliday(new Holiday('jeuneGenevois', [
+            'en' => 'Jeûne genevois',
```

Comment:
Not sure why you've added the English element just with the French translation. Is it because there is no English translation? In that case, you can leave out the English one as Yasumi will always pick the native language.

Reply:
Indeed, it is unneeded. I've removed both English translations.